<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartWindow</class>
 <widget class="QMainWindow" name="chartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pBtn_leftMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_rightMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Legend</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>resources/icons/arrow_left.png</normaloff>resources/icons/arrow_left.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedW_left">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_axis">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>66</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>AXIS</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>171</width>
            <height>201</height>
           </rect>
          </property>
          <property name="title">
           <string>X Axis</string>
          </property>
          <widget class="QComboBox" name="cBox_xAxis">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>86</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>100</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>scale</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_x_start">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>70</y>
             <width>44</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_x_stop">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>110</y>
             <width>44</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_x_scale">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>150</y>
             <width>44</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>290</y>
            <width>171</width>
            <height>201</height>
           </rect>
          </property>
          <property name="title">
           <string>Y Axis</string>
          </property>
          <widget class="QComboBox" name="cBox_yAxis">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>151</width>
             <height>26</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Total sended codewords</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nak</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ack</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Retransmitted words</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Failed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Not detected corrupted words</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Uncorrupted words detected as corrupted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Correct sended words</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Retransmitted bits </string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>160</y>
             <width>66</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>scale</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_y_scale">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>150</y>
             <width>111</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_y_start">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>70</y>
             <width>101</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="sBox_y_stop">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>110</y>
             <width>111</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>66</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_series">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Available &lt;/p&gt;&lt;p&gt;Data Series&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listW_availableSeries">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Double click on data series, to add it to the chart</string>
            </property>
            <property name="statusTip">
             <string>Double click on data series, to add it to the chart</string>
            </property>
            <property name="whatsThis">
             <string>bleh</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="widgetChart">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedW_right">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>66</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Legend</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLegend_Menu"/>
    <addaction name="actionAxis_Menu"/>
    <addaction name="actionData_Series"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLegend_Menu">
   <property name="text">
    <string>Legend Menu</string>
   </property>
  </action>
  <action name="actionAxis_Menu">
   <property name="text">
    <string>Axis Menu</string>
   </property>
  </action>
  <action name="actionData_Series">
   <property name="text">
    <string>Data Series</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
